### PR TITLE
ARTEMIS-1276 avoid System.exit in tests as Surefire cannot deal with that

### PR DESCRIPTION
**Pull request by jdanekrh**
This change results in many more tests being run. Previously, 662 tests
were executed. Now, 1166 tests is executed. The running time has increased
from ~20 minutes to ~50 minutes.

Before

> Tests run: 662, Failures: 14, Errors: 4, Skipped: 1

After

> Tests run: 1166, Failures: 38, Errors: 1, Skipped: 2

(the second commit in the PR disables one of the erroring tests, because it tests feature that Artemis does not implement)

### CODE REVIEW

File: tests/activemq5-unit-tests/pom.xml [hunk 1]
```diff
@@ -460,6 +460,8 @@
                   <exclude>**/org/apache/activemq/ProducerFlowControlTest.java</exclude>
                   <!-- exclude tests that are on client side only -->
                   <exclude>**/org/apache/activemq/transport/tcp/TransportConnectorInvalidSocketOptionsTest.java</exclude>
+                  <!-- exclude tests for features not implemented in Artemis -->
+                  <exclude>**/org/apache/activemq/MessageEvictionTest.java</exclude>
```
Comment (clebertsuconic) [inline tests/activemq5-unit-tests/pom.xml]: why not just remove the test?

Comment (jdanekrh) [inline tests/activemq5-unit-tests/pom.xml]: if that is how it is done regarding non-applicable activemq5 tests in Artemis, then sure, I can just delete the test.

**Comment by clebertsuconic** [comment]
Just remove the test then please ?

File: tests/activemq5-unit-tests/pom.xml (hunk 1 above)
Comment (jdanekrh) [inline tests/activemq5-unit-tests/pom.xml]: done

**Pull request closed without merging**